src/top/vision/shortcut.cc

```diff
@@ -0,0 +1,37 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file shortcut.cc
+ * \brief Property def of pooling operators.
+ */
+#include <nnvm/op.h>
+#include <nnvm/node.h>
+#include <nnvm/op_attr_types.h>
+#include <nnvm/top/nn.h>
+#include "../op_common.h"
+#include "shortcut.h"
+
+namespace nnvm {
+namespace top {
+
+NNVM_REGISTER_OP(shortcut)
```

Comment:
yolo2_shortcutt

Reply:
shortcut is not used for yolo2.. its common operator, used for resnet.

Reply:
I double checked the implementation of shortcut, and it appeared to be a generalization of elemwise add (when two input's shape and channel differ). For standard operator in resnet, usually elemwise_add is used, instead of shortcut.

Since this is non-conventional, we should either do the on the fly conversion(by checking shape equals and translate to elemwise_add, or use a special name(darknet_shortcut). I prefer translation to elemwise_add when possible so nnvm graph is more canonicalized

@PariksheetPinjari909 what do you think

Reply:
There can be 3 scenarios for shape diff, a=b, a<b, a>b.  in some cases we cannot do elementwise add. so i added the shortcut implementation.. I also suggest  "darknet_shortcut" will be the right name for this case.

Reply:
style and name changes, i will update and update this PR soon.